src/offset_of.rs

```diff
@@ -67,6 +67,28 @@ macro_rules! _memoffset_offset_from_unsafe {
         ($field as usize) - ($base as usize)
     };
 }
+#[cfg(not(feature = "unstable_offset_of"))]
+#[macro_export(local_inner_macros)]
+#[doc(hidden)]
+macro_rules! _memoffset__offset_of_impl {
```

Comment:
What's the reasoning for introducing another layer of macros instead of `cfg`-ing `offset_ot`?

Related to the doc-comments, or something else?

Reply:
Yeah I didn't know of another way to share the doc comments. There are ways to move doc comments out to a different file, but those were introduced in 1.54, so way after 1.19.0.